app/helpers/sessions_helper.rb

```diff
@@ -16,6 +36,7 @@ def logged_in?
 
   # Logs out the current user.
   def log_out
+    forget(current_user)
```

Comment:
```suggestion
    forget current_user
```
ruby k khuyến khích dùng ngoặc. chỗ nào bỏ dc thì bỏ nhé e